src/KristofferStrube.Blazor.FileSystemAccess/FileSystemWritableFileStream.InProcess.cs

```diff
@@ -13,7 +13,7 @@ public class FileSystemWritableFileStreamInProcess : FileSystemWritableFileStrea
 
     public static async Task<FileSystemWritableFileStreamInProcess> CreateAsync(IJSRuntime jSRuntime, IJSInProcessObjectReference jSReference)
```

Comment:
We should add a creator method to the `IFileSystemAccessService` for the `FileSystemWritableFileStreamInProcess` as we have done with the handles and obsolete this as we have done for the others.